File: 삼성/1926.py
n = int(input())
for i in range(1, n + 1):
    num = i
    clap = 0
    while num:
        digit = num % 10
        if digit and digit % 3 == 0:
            clap += 1
        num //= 10
        
    print("-" * clap, end=' ') if clap else print(i, end=' ')

File: SW Expert Academy/1926.cpp
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    for (int i = 1; i <= n; i++) {
        int num = i;
        int clap = 0;
        while (num) {
            int digit = num % 10;
            if (digit % 3 == 0 && digit) clap++;
            num /= 10;
        }
        
        if (clap) {
            for (int j = 0; j < clap; j++) printf("-");
        }
        else printf("%d", i);
        printf(" ");
    }

    return 0;
}